### ISSUE TITLE
Apply Custom Filter?

### ISSUE THREAD

**Issue by SohailYaseen** [nekocode/CameraFilter#20]
I want to apply my custom filters like Vignette, GrayScale, Blur, Blue Mess and many more, how can i achieve using your code.?

**Comment by nekocode** [comment]
You can take a look at the [PixelizeFilter.java](https://github.com/nekocode/CameraFilter/blob/master/app/src/main/java/cn/nekocode/camerafilter/filter/PixelizeFilter.java) to learn how to make a simple filter.

**Comment by SohailYaseen** [comment]
It requires .fsh file, and that's the reason i am asking this question. How can i write .fsh file of my custom filter! @nekocode

**Comment by nekocode** [comment]
Yes, you need to write the fragment shader for your custom filter. This is a difficult science. Maybe you need to learn more about the OpenGL and shader programing.

If you need some examples. Just check the links in the README that link to the shadertoy.com.
